dbt/include/athena/macros/materializations/seeds/helpers.sql

```diff
@@ -12,6 +12,7 @@
   {%- set quote_seed_column = model['config'].get('quote_columns', None) -%}
   {%- set s3_data_dir = config.get('s3_data_dir', default=target.s3_data_dir) -%}
   {%- set s3_data_naming = model['config'].get('s3_data_naming', target.s3_data_naming) -%}
+  {%- set external_location = config.get('external_location', default=none) -%}
```

Comment:
The external_location is at the seed level, not the project level right ?

If so I am not that it works, shouldn't it be : 

```sql
{%- set external_location = model['config'].get('external_location', default=none) -%}
```

Reply:
What's the difference? I see `s3_data_dir` being set from `config` and `s3_data_naming` from `model`. 
In my case, I added an `external_location` as a seed config to a property file, which worked.

Reply:
Can you send me your yaml config file for your seed so I can understand how you set the external location please ?

Reply:
In `seeds/some_schema/`, I have a `_seed_some_schema__models.yml` file with some basic config like so:
```yaml
version: 2

seeds:
  - name: some_super_seed
    config:
      external_location: "s3://s3_bucket/some_schema/some_super_seed"
      column_types:
        col1: varchar(64)
        col2: varchar(64)

```

along with the actual `some_super_seed.csv` file

Reply:
this {%- set external_location = config.get('external_location', default=none) -%} should work too, they are equivalent.

Reply:
Actually for consistency also {%- set s3_data_naming = model['config'].get('s3_data_naming', target.s3_data_naming) -%} should be changed to use simply config.get.
I believe that model['config'] is equivalent to config.

Reply:
what about the other variables (`column_override`, `quote_seed_column`). As `model` is an input arg, is it possible it might be used somewhere where there is no `config` in the namespace?

Reply:
I think that is the same, you can try to change that too, and see if you get the expected behavior. I believe that are all use in the context of seeding.